examples/alert_triage_agent/pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools >= 64", "setuptools-scm>=8"]
+
+[tool.setuptools_scm]
+root = "../.."
+
+[project]
+name = "aiq_alert_triage_agent"
+dynamic = ["version"]
+dependencies = [
+  "aiqtoolkit[langchain]",
+  "langchain-core>=0.1.0",
+  "pandas>=2.0.0",
+  "ansible-runner>=2.3.0",
+  "langgraph>=0.0.10",
+  "flask>=3.0.0"
+]
+requires-python = ">=3.12"
```

Comment:
`aiqtoolkit` supports Python 3.11 and 3.12, unless this example explicitly needs something in 3.12, I would change this to match the project.